package.json

```diff
@@ -8,8 +8,7 @@
   },
   "scripts": {
     "test": "./test/runtests.sh",
-    "compile": "rm -rf lib/ && ./node_modules/.bin/babel -sd lib/ src/",
-    "pretest": "npm run compile",
```

Comment:
So, the only reason to use this is to make sure the target works.

We have to use mocha.opts with `--compilers js:babel/register` or the test file itself would have to be compiled...

Note, `--compilers js:babel/register` is not the same as `babel-node` it doesn't start a subprocess, just overloads the `require` function (I think)... I haven't had issues with it so far, I also (hope) it caches a little bit... But our compile step certainly doesn't cache anything, so...

I think there's value to not dealing with compile for the tests, as we need to use this for loading the test file anyways.